Java/PoolAndResourceFile/src/main/java/PoolAndResourceFile.java

```diff
@@ -356,7 +356,7 @@ public static void main(String argv[]) throws Exception {
 
             if (shouldDeletePool) {
                 try {
-                    client.jobOperations().deleteJob(poolId);
+                    client.poolOperations().deletePool(poolId);;
```

Comment:
Thanks for spotting this, @sho-yamashita! The change looks good, but there's an extra semi-colon at the end of the line. Mind fixing that?

Reply:
@dpwatrous 
sorry i'm fix it.